homeassistant/components/squeezebox/media_player.py

```diff
@@ -460,7 +466,9 @@ async def async_play_media(self, media_type, media_id, **kwargs):
             media_id = play_item.url
 
         if media_type in MEDIA_TYPE_MUSIC:
-            media_id = async_process_play_media_url(self.hass, media_id)
+            if not media_id.startswith(SQUEEZEBOX_SOURCE_STRINGS):
+                # do not process special squeezebox "source" media ids
+                media_id = async_process_play_media_url(self.hass, media_id)
 
             await self._player.async_load_url(media_id, cmd)
```

Comment:
Ah my bad. I didn't realize that `async_load_url` would also be used with non-urls